neuralprophet/forecaster.py

```diff
@@ -825,6 +850,9 @@ def split_df(self, df, freq="auto", valid_p=0.2, local_modeling=False):
             inputs_overbleed=True,
             local_modeling=local_modeling,
         )
+        if local_modeling:
```

Comment:
Yes, the idea is if the user passes a dict we should return a dict. In case of a list, we return a list and so on. However, in case we receive a dict and do not do a local_split then it becomes a little bit more complicated. I will think about this.